### ISSUE TITLE
Issues with watchOS 7 Betas

### ISSUE THREAD

**Issue by longzheng** [Sunoo/homebridge-camera-ffmpeg#635]
**Describe The Bug:**
I'm not seeing any snapshots or video streams in watchOS 7 (both developer beta 1 and beta 2). Everything is working in iOS 14 Home app or macOS Home app.

Not sure if issue with `homebridge` or `homebridge-camera-ffmpeg` or watchOS 7, just wanted to see if anyone else experiencing issues. But because I have other homebridge plugins working in watchOS 7, I'm inclined to think it is specific to `homebridge-camera-ffmpeg`.

**To Reproduce:**
- Open Home app on watch
- Scroll to see a camera, wait for snapshot, no image
![image](https://user-images.githubusercontent.com/484912/87144592-5953f900-c2eb-11ea-9f2e-918da1ddb7e6.png)

- Tap a camera to view stream, stuck at "Connecting..."
![image](https://user-images.githubusercontent.com/484912/87144604-5d801680-c2eb-11ea-8150-0d6435fc2485.png)

**Expected behavior:**

**Logs:**
When I just browse Home, I see requests for snapshot in the debug but nothing appears on screen.
```
[10/07/2020, 8:18:10 pm] [Camera-ffmpeg] Snapshot from Backyard at 320:240
[10/07/2020, 8:18:10 pm] [Camera-ffmpeg] ffmpeg -rtsp_transport tcp -re -i rtsp://... -frames:v 1 -vf scale=320:240 -f image2 -
```

When I tap camera to view video, I don't see any logs come through for stream start (e.g. `Starting Front door video stream`). Weirdly if I tap back, I see some a log for stream end.
```
[10/07/2020, 8:10:39 pm] [Camera-ffmpeg] Stopped Front door video stream!
```

**Homebridge Config:**

```json
        {
            "cameras": [
                {
                    "name": "Front door",
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -re -i rtsp://...",
                        "maxFPS": 20,
                        "debug": true
                    }
                },
```

**Screenshots:**

**Environment:**

* **Node.js Version**: v14.5.0
* **NPM Version**: 6.14.6
* **Homebridge Version**: 1.2.0-beta.29
* **Homebridge Camera FFmpeg Version**: 2.1.1
* **Homebridge Config UI X Plugin Version**: v4.23.2
* **Operating System**: Windows

**Comment by Sunoo** [comment]
I never use the camera from my Watch, but I just confirmed that my Series 4 running watchOS7 beta 2 has the same issue.

**Comment by longzheng** [comment]
I wonder if someone has a HomeKit native camera that can confirm if it's an issue with watchOS 7 or homebridge.

**Comment by Sunoo** [comment]
For what it’s worth, my Ring doorbell (through a different Homebridge plugin) also behaves this same way on my Watch. Unfortunately I do not have access to a native HomeKit camera to test with at this time.

However, I spent some time playing with the Home app on the Watch, and it seems to be pretty flaky in the current beta. Several times I had to restart the Watch in order to get any of my devices to respond in it.

Unless someone gives evidence otherwise (testing with a native camera would be ideal), I’m currently assuming this is a watchOS issue.

**Comment by longzheng** [comment]
I just tested https://github.com/homebridge/homebridge-examples/tree/master/bridged-camera-example-typescript and experienced the same issue on watchOS so inclined to rule out this plugin specifically. 

Still a possibility that issue is homebridge, but more likely watchOS 7 beta.

**Comment by longzheng** [comment]
I think I figured out the problem. (I'm not sure if the problem is a bug with watchOS or homebridge).

I had to manually connect to my Wi-Fi using the watchOS settings. Whilst most of the watchOS functionality and the Home app on the watch seems to work with just a phone/Bluetooth connection, after I connected Wi-Fi the camera snapshots and video streams started working in my Home app.

![image](https://user-images.githubusercontent.com/484912/91239663-8a956500-e783-11ea-88b3-b189307f8643.png) ![image](https://user-images.githubusercontent.com/484912/91239665-8cf7bf00-e783-11ea-8573-21a6d5ad711d.png)
